analyzers/tests/SonarAnalyzer.UnitTest/TestCases/InitializationVectorShouldBeRandom.cs

```diff
@@ -260,6 +257,17 @@ public void ImplicitlyTypedArrayNonByte()
             }
         }
 
+        public void CollectionInitializer()
+        {
+            List<byte> listWithConstant = new List<byte> { 0x00 };
+            List<byte> list = new List<byte> { Rnd() };
+            using (AesCryptoServiceProvider aes = new AesCryptoServiceProvider())
+            {
+                ICryptoTransform encryptor = aes.CreateEncryptor(aes.Key, listWithConstant.ToArray()); // FN
```

Comment:
I'll send an email to @pierre-loup-tristant-sonarsource  about this FN. 
Let's keep the scope of the PR limited and only fix the initial ticket.

Reply:
follow-up https://github.com/SonarSource/sonar-dotnet/issues/4565